### PR TITLE
added recipe for testfixtures

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
built fine for me on py2.7 and py3.5

### CODE REVIEW

File: recipes/testfixtures/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+    home: http://pythonhosted.org/testfixtures/
+    license: MIT License
+    summary: 'A collection of helpers and mock objects for unit tests and doc tests.'
+    doc_url: http://pythonhosted.org/testfixtures/
+
+extra:
+    recipe-maintainers:
+        # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: This can be dropped.

File: recipes/testfixtures/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    run:
+        - python
+
+test:
+    imports:
+        - testfixtures
+
+about:
+    home: http://pythonhosted.org/testfixtures/
+    license: MIT License
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: Please drop "License" as it is redundant.

File: recipes/testfixtures/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: Please drop this comment.

File: recipes/testfixtures/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    home: http://pythonhosted.org/testfixtures/
+    license: MIT License
+    summary: 'A collection of helpers and mock objects for unit tests and doc tests.'
+    doc_url: http://pythonhosted.org/testfixtures/
+
+extra:
+    recipe-maintainers:
+        # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+        - ChrisBarker-NOAA
+        - JamesMakela-NOAA
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: @JamesMakela-NOAA, are you ok being added as a maintainer on `testfixtures`?

File: recipes/testfixtures/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+about:
+    home: http://pythonhosted.org/testfixtures/
+    license: MIT
+    summary: 'A collection of helpers and mock objects for unit tests and doc tests.'
+    doc_url: http://pythonhosted.org/testfixtures/
+
+extra:
+    recipe-maintainers:
+        - ChrisBarker-NOAA
+        - JamesMakela-NOAA
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: @JamesMakela-NOAA, are you ok being added as a maintainer on `testfixtures`?

File: recipes/testfixtures/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - testfixtures
+
+about:
+    home: http://pythonhosted.org/testfixtures/
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: Seems this has been changed to ( https://github.com/Simplistix/testfixtures ) according to [PyPI]( https://pypi.python.org/pypi/testfixtures/4.10.0 ).

File: recipes/testfixtures/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    license: MIT
+    summary: 'A collection of helpers and mock objects for unit tests and doc tests.'
+    doc_url: http://pythonhosted.org/testfixtures/
```
Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: PyPI says this. Though they have [readthedocs]( http://testfixtures.readthedocs.io/en/latest/ ) too. Not sure what we should prefer.

Comment (jakirkham) [inline recipes/testfixtures/meta.yaml]: Added this issue ( https://github.com/Simplistix/testfixtures/issues/45 ) to get devs' opinions.

**Comment by ChrisBarker-NOAA** [comment]
Thanks Filipe! -- I"ll go in and update the doc link to read the docs at some point, once its in a feedstock.

**Pull request merged**